edb/testbase/server.py

```diff
@@ -1092,19 +1092,16 @@ async def setup_and_connect(cls):
             # The retry here allows the test to survive a concurrent testing
             # EdgeDB server (e.g. async with tb.start_edgedb_server()) whose
             # introspection holds a lock on the base_db here
-            async def create_db():
-                async for tr in cls.try_until_succeeds(
-                    ignore=edgedb.ExecutionError,
-                    timeout=30,
-                ):
-                    async with tr:
-                        await admin_conn.execute(
-                            f'''
-                                CREATE DATABASE {qlquote.quote_ident(dbname)}
-                                FROM {qlquote.quote_ident(base_db_name)}
-                            ''',
-                        )
-            await create_db()
+            create_command = f'CREATE DATABASE {qlquote.quote_ident(dbname)}'
```

Comment:
```suggestion
            create_command = f'CREATE BRANCH {qlquote.quote_ident(dbname)}'
```

Reply:
I went to try to fix this in the web UI because I was in the middle of something else locally and I wound up in a full vscode and terminal environment in my browser. Fun.